README.md

```diff
@@ -215,6 +215,37 @@ You can also set specific database connection to be used by adding `--database`
 ```php
 $this->loadLaravelMigrations(['--database' => 'testbench']);
 ```
+
+#### Runing Testing Migrations
+To run migrations that are only used for testing purposes and not part of your package, add the following to your base test class:
+
+```php
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->loadMigrationsFrom(__DIR__ . '/database/migrations');
+        
+        // and other test setup steps you need to perform
+    }
+
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function getPackageProviders($app)
+    {
+        return [
+            // your package service provider,
+            Orchestra\Database\ConsoleServiceProvider\ConsoleServiceProvider::class,
+        ];
+    }
+```
+
+##### Notes and Considerations
+- Your migration files has to suite Laravel's convention, e.g. 0000_00_00_000000_create_package_test_tables.php
+- You may choose to put your migrations folder in tests/database/
+- You may just copy the migrations you're going to publish into your dedicated test-migration directory
```

Comment:
I don't agree with this line.

1. Laravel already offer publishing migrations command, should be able to run the command and migrate normally.
2. Laravel also allows to register custom migration paths, which render this as redundant.